### ISSUE TITLE
cannot run steempy

### ISSUE THREAD

**Issue by firemouse604** [steemit/steem-python#178]
When I try to run steempy from CLI this is what I get:

```
Traceback (most recent call last):
  File "/usr/local/bin/steempy", line 7, in <module>
    steem.cli.legacy()
  File "/usr/local/lib/python3.6/site-packages/steem/cli.py", line 83, in legacy
    version=pkg_resources.require("steem")[0].version
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 984, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 875, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (toml 0.9.3 (/usr/local/lib/python3.6/site-packages), Requirement.parse('toml==0.9.3.1'), {'steem'})
```

**Comment by bobinson** [comment]
Its an error with the toml version & the fix is discussed in #176

**Comment by cyon1c** [comment]
Duplicate of #176.
